src/core_plugins/kibana/translations/en.json

```diff
@@ -1,4 +1,32 @@
 {
   "UI-WELCOME_MESSAGE": "Loading Kibana",
-  "UI-WELCOME_ERROR": "Kibana did not load properly. Check the server output for more information."
+  "UI-WELCOME_ERROR": "Kibana did not load properly. Check the server output for more information.",
+  "KIBANA-H1_CONFIGURE_INDEX_PATTERN": "Configure an index pattern",
+  "KIBANA-MUST_CONFIGURE_INDEX_PATTERN": "In order to use Kibana you must configure at least one index pattern. Index patterns are used to identify the Elasticsearch index to run search and analytics against. They are also used to configure fields.",
+  "KIBANA-CONTAINS_TIME_BASED_EVENTS_CHECKBOX": "Index contains time-based events",
+  "KIBANA-INDEX_NAME_IS_PATTERN_CHECKBOX": "Use event times to create index names <small>[DEPRECATED]</small>",
```

Comment:
Ok, have moved strings around and tags only now in the HTML. Let me know what you think.